resources/views/campaign-ids/show.blade.php

```diff
@@ -10,11 +10,18 @@
                 <h3>Internal Campaign Name</h3>
                 <p>{{ $campaign->internal_title }}</p>
 
+                <h3>Campaign ID</h3>
+                <p>{{ $campaign->id }}</p>
+
```

Comment:
this is great! This actually was a suggestion from when we went through with Adam that this would be a nice to have - thank you for adding!